### ISSUE TITLE
Unsupported zones

### ISSUE THREAD

**Issue by kvharish** [moment/luxon#958]
**Describe the bug**
Below Zones are currently considered as invalid by Luxon. Is there any plan to support these?

PST8PDT
EST5EDT
GMT0
MST7MDT
CST6CDT

**To Reproduce**
When I use any of the above 5 zones in the below code snippet I am getting "unsupported zone" error.

`luxon.DateTime.fromMillis(1623830102).setZone('GMT0')`

Luxon version [e.g. 1.25.0] 1.27.0

**Comment by icambron** [comment]
Yeah, what's happening here is that Luxon doesn't think those are valid IANA zone identifiers because they lack the slash (e.g. `Etc/GMT0" is valid, though it doesn't look Intl recognizes all of those if you add a leading `Etc/` to them. Possibly, Luxon could be told to support those really explicitly.

**Comment by kvharish** [comment]
Hi thanks for taking your time to reply. Using Etc/ works only for GMT0 for the other zone identifiers they are still unsupported.

**Comment by peabnuts123** [comment]
I have a few years of production data from a site that used moment.js, so I have a few records with some of these old timezones laying around that need migrating. 

In my case I can use a simple rewrite rule when I migrate the data:
```javascript
const TIMEZONE_UPDATE_RULES = {
  'PST8PDT': 'US/Pacific',
  'EST5EDT': 'US/Eastern',
  'CST6CDT': 'US/Central',
  'MST7MDT': 'US/Mountain',
  'GMT+0': 'Etc/GMT+0',
  'GMT0': 'Etc/GMT0',
};
```

but I acknowledge that this is not the best solution. As @kvharish mentioned above, these timezones are not supported in any way as far as I can tell (e.g. `Etc/PST8PDT` is not valid). Is there something that can be done to support these missing/"deprecated" timezones? Perhaps they could log a warning or something when they are used, stating their modern equivalent.

**Comment by icambron** [comment]
This is fixed in 7881b750 and will be in the next release (likely 2.4.0)
